wagtail_localize/static_src/src/editor/components/TranslationEditor/index.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { FunctionComponent } from 'react';
+
+import Section from '../../../common/components/Section';
+import {Tabs, TabContent} from '../../../common/components/Tabs';
+
+import { EditorState, reducer } from './reducer';
+import EditorHeader from './header';
+import EditorFooter from './footer';
+import EditorSegmentList from './segments';
+import EditorToolbox from './toolbox';
+
+export interface User {
+    full_name: string;
+    avatar_url: string | null;
+}
+
+export interface Locale {
+    code: string;
+    displayName: string;
+}
+
+export interface BreadcrumbItem {
+    isRoot: boolean;
+    title: string;
+    exploreUrl: string;
+}
+
+export interface Translation {
+    title: string;
+    locale: Locale;
+    editUrl?: string;
+}
+
+export interface StringSegment {
+    id: number;
+    contentPath: string;
+    source: string;
+    location: {
+        tab: string;
+        field: string;
+        blockId: string|null;
+        subField: string | null;
+        helpText: string;
+    };
+    editUrl: string;
+}
+
+export interface StringTranslationAPI {
+    string_id: number;
+    segment_id: number;
+    data: string;
+    error: string;
+    comment: string;
+    last_translated_by: User | null;
```

Comment:
Yeah, I see what you mean. I started out with JS naming conventions for everything but we use python here because this interface is used in the API which use DRF serializers.